# HG changeset patch
# Parent 059a1d81b9fb4fb9ca5069ebe7677a3e7f07e0fc

No banlimit for bans set with OPMODE.

Backport from ircu - http://sourceforge.net/p/undernet-ircu/undernet-ircu/ci/8fc59ec8ed68be82ca9314ac95a093a73623b882/

diff -r 059a1d81b9fb ircd/channel.c
--- a/ircd/channel.c
+++ b/ircd/channel.c
@@ -3003,6 +3003,7 @@
 	len -= banlen;
       } else {
 	if (state->flags & MODE_PARSE_SET && MyUser(state->sptr) &&
+            !(state->mbuf->mb_dest & MODEBUF_DEST_OPMODE) &&
 	    (len > (feature_int(FEAT_AVBANLEN) * feature_int(FEAT_MAXBANS)) ||
 	     count > feature_int(FEAT_MAXBANS))) {
 	  send_reply(state->sptr, ERR_BANLISTFULL, state->chptr->chname,
